src/DashboardWidgetItem.vue

```diff
@@ -211,18 +209,34 @@ export default {
 	},
 
 	props: {
-		item: {
-			type: Object,
+		targetUrl: {
```

Comment:
from the template I would guess that this should be `require: true`, and any other mandatory props as well

Reply:
Thanks for the review!

I'd like to keep this as flexible as possible. The only mandatory prop is the main text imho.

I adapted the default values:
* targetUrl: undefined so the link's href will not be defined (it used to be an empty string so the link pointed to the dashboard page itself...)
* avatarUrl, avatarUsername and overlayIconUrl: undefined
* subText: empty string

Reply:
IMO we shouldn't use a `<a>` if it's not a link. It will trigger an anchor navigation to `#`. Let's just use a `span` or `div` for non-links.


Fine to have everything else optional :)